ArigatouApp/Presenter/SpeechPresenter.swift

```diff
@@ -160,7 +162,18 @@ class SpeechPresenter{
 }
 
 extension SpeechPresenter: PresenterInput {
+    
     func viewDidLoad() {
+        if matchCountManger.getCount() >= MAX_COUNT {
```

Comment:
```suggestion
        let shouldShowEndScreen = matchCountManger.getCount() >= MAX_COUNT
        //  もしくは let hasReachedMaxCount = matchCountManger.getCount() >= MAX_COUNT
        if shouldShowEndScreen {
```

[nits] 条件文 -> 変数に切り出し